### PR TITLE
Refactor all remaining code

### PR DESCRIPTION
**Pull request by stanley-1**
ALL references to `address book` have been changed to `socialite` or `contact list`

### CODE REVIEW

**Comment by stanley-1** [comment]
Thanks @bnjmnt4n and @Zacchua for doing a great job with refactoring `Storage`, allowing the remaining changes to be smoother! 😄

**Review by david-eom — APPROVED** [review]
Didn't actually look through every single line, just roughly skimmed through, but LGTM!

File: src/test/java/socialite/testutil/TypicalPersons.java [hunk 1]
```diff
@@ -84,14 +84,14 @@
     private TypicalPersons() {} // prevents instantiation
 
     /**
-     * Returns an {@code ContactList} with all the typical persons.
+     * Returns a {@code ContactList} with all the typical persons.
      */
-    public static ContactList getTypicalAddressBook() {
-        ContactList ab = new ContactList();
+    public static ContactList getTypicalContactList() {
+        ContactList cl = new ContactList();
```
Comment (david-eom) [inline src/test/java/socialite/testutil/TypicalPersons.java:90]: Great one!

**Review by Zacchua — APPROVED** [review]
Great work Stanley! You have amazing attention to detail!

**Pull request merged**